linux-anvil-aarch64/Dockerfile

```diff
@@ -13,11 +13,18 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE=en_US.UTF-8
 
 # bust the docker cache so that we always rerun the installs below
-ADD http://www.randomtext.me/api/gibberish /opt/docker/etc/gibberish
+ADD https://loripsum.net/api /opt/docker/etc/gibberish
```

Comment:
Can we please make this change in the other `Dockerfile`s? They will likely encounter the same issue

Reply:
Yes, I will do this in another PR for an easier review

Reply:
Addressed in https://github.com/conda-forge/docker-images/pull/183

Reply:
Thanks! That's now merged. Could we please rebase this?

Reply:
Done!